appf.adoc

```diff
@@ -112,6 +112,47 @@ Note also that the values "x" and "y" are not case-sensitive.
 The use of `projection_x_coordinate` and `projection_y_coordinate` was deprecated in version 1.9 of the CF Conventions.
 The initial definition of this projection used these standard names to identify the projection coordinates even though their canonical units (meters) do not match those required for this projection (radians).
 
+[[healpix]]
+=== HEALPix
+
+----
+grid_mapping_name = healpix
+----
+
+__Map parameters:__::
+* **`indexing_scheme`** - The HEALPix indexing scheme: One of **`nested`**, **`ring`**, **`nuniq`**, or **`zuniq`**
+* **`refinement_level`** - The HEALPix refinement level: Mandatory for indexing scheme **`nested`** or **`ring`**, but must be omitted for indexing scheme **`nuniq`** or **`zuniq`**
+
+__Map coordinates:__::
+The integer-valued HEALPix index coordinates are identified by the **`standard_name`** attribute value **`healpix_index`**.
+The HEALPix index coordinates may be stored in a coordinate, auxiliary, or scalar coordinate variable. Indexing schemes are optimized to allow applications efficient data access when the HEALPix indices are ordered strictly monotonically. Therefore it is recommended to store strictly monotonic HEALPix indices in a coordinate variable, which allows an application to assess their monotonicity without expensive checking.
++
+The coordinate system is defined by discrete cells on a reference ellipsoid (or sphere), arranged on a HEALPix grid.
+Each cell is represented by its cell index, which, together with the `indexing_scheme` and possible `refinement_level`, determines center coordinates (latitude and longitude) as well as boundaries.
+See <<GHB05>> for the calculation of center coordinates and cell boundary coordinates.
+
+__Notes:__::
+
+Cells are indexed using the scheme identified by the **`indexing_scheme`** parameter.
++
+For both the **`nested`** and **`ring`** schemes, the grid resolution is controlled by the refinement level parameter **`refinement_level`** <<GHB05>>.
+The total number of cells on the surface of the sphere (or reference ellipsoid) is `12 * (4 ^ refinement_level)`.
+The lowest allowed refinement level is `0`, giving 12 cells globally.
+The algorithm that assigns cell locations to **`nested`** or **`ring`** indices at a given refinement level is defined in <<GHB05>>. In this algorithm, a cell with index `0`, at any refinment level, always has its southern vertex on the meridian at 45 degrees east.
```

Comment:
```suggestion
The algorithm that assigns cell locations on a sphere to **`nested`** or **`ring`** indices at a given refinement level is defined in <<GHB05>>.
In this algorithm, a cell with index `0`, at any refinement level, always has its southernmost vertex on the meridian at 45 degrees east.
Cell locations on the surface of an ellipsoid are defined to give equal-area HEALPix cells when using authalic latitudes (which map locations on an ellipsoid to those on the sphere with the same surface area).
For instance, on the WGS84 ellipsoid this results in the center of the HEALPix cell at refinement level `0` with index `0` being at geodetic latitude 41.938 degrees north, instead of at 41.810 degrees north in the case of a sphere, whilst the longitude is at 45 degrees east in both cases.
```